base/docs/basedocs.jl

```diff
@@ -644,6 +644,31 @@ desired can be used.
 """
 kw"if", kw"elseif", kw"else"
 
+"""
+    a ? b : c
+
+Short form for conditionals; read "if `a`, evaluate `b` otherwise evaluate `c`".
+Also known as the
+[ternary operator](https://docs.julialang.org/en/v1/manual/control-flow/#man-conditional-evaluation-1).
```

Comment:
```suggestion
```

Reply:
By the way, multi-line suggestions now work in the github code review! (Just click and drag down over the lines you want to make into a suggestion.)